### DISCUSSION TITLE
Porque consideram testes unitários no frontend importantes?

### DISCUSSION THREAD

**Discussion by felquis** [frontendbr/forum#2304] [Discussão]
Oi, acompanhando a discussão https://github.com/frontendbr/forum/discussions/1169 "Por que não estamos fazendo testes de unidade no frontend?" que é uma pergunta interessante.

[Apareceu uma outra pergunta](https://github.com/frontendbr/forum/discussions/1169#discussioncomment-2846173) que acho que sintetiza bem a discussão e gostaria de extende-la para um novo tópico:

**Comment by RatoX** [comment]
É esperado que uma função que receba N parâmetros retorne o valor X. Como códigos evoluem absurdamente rápido, precisamos garantir uma estabilidade mínima através do uso de testes unitários.

Não importa se é frontend ou backend. Se tem uma função, um parâmetro, um resultado ou um side-effect. Teste.
